### ISSUE TITLE
Roller shade group from ESPSomfy unavailable with HA integration v.2.1.9

### ISSUE THREAD

**Issue by srkoster** [rstrouse/ESPSomfy-RTS-HA#25]
I've combined two roller shades into a group using the ESPSomfy web application and integrated them into Home Assistant using the integration. All was working well in v.2.1.6. but after upgrading the integration to v.2.1.9. the group entity became unavailable.

I was running my ESPSomfy on 2.1.6 and tried upgrading to 2.1.9 but failed as I couldn't find the S2 version (the artifact was expired), so I upgraded to 2.1.10. However, this didn't change the situation.

Now I've rolled back my HA integration to v.2.1.8. and the group entity is working again.

With v.2.1.9. I see this in my device page in Home Assistant (it's the "Slaapkamer" entity). The screenshot is before the ESPSomfy upgrade, but as stated, that had no effect.
![image](https://github.com/rstrouse/ESPSomfy-RTS-HA/assets/89455157/5146b09f-9540-42f8-873a-5380c0d9415f)

Back to v.2.1.8. shows the controls again
![image](https://github.com/rstrouse/ESPSomfy-RTS-HA/assets/89455157/386b5906-ca6a-4df9-9e48-238078fd710b)

Also, with v.2.1.9. I tried controlling the grouped shades using my HA cover card, resulting in this error message
```
2023-10-17 10:44:55.595 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [547918101056] Entity cover.slaapkamer does not support this service.
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 227, in handle_call_service
    await hass.services.async_call(
  File "/usr/src/homeassistant/homeassistant/core.py", line 1969, in async_call
    response_data = await coro
                    ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 2006, in _execute_service
    return await target(service_call)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 235, in handle_service
    return await service.entity_service_call(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 851, in entity_service_call
    raise HomeAssistantError(
homeassistant.exceptions.HomeAssistantError: Entity cover.slaapkamer does not support this service.
```

**Comment by rstrouse** [comment]
I'll have a look.  Are these tilt only blinds by chance?

**Comment by srkoster** [comment]
Thanks for the quick response. These are zipscreens which only have up and down control, no tilt. They are defined as a roller shade in ESPSomfy.

**Comment by rstrouse** [comment]
Actually, HA is no longer registering the entities that are sent to the group since the latest update.  I'll dig to see what has changed.

**Comment by rstrouse** [comment]
Alright I published v2.1.10 of the integration.  The HA folks changed the attributes for the CoverGroup which essentially trashed the group.  Update to v2.1.10 and your troubles will go away.

**Comment by srkoster** [comment]
Thanks for the quick fix! Everything is working again :)
